File: CS-Examples/18_Tables/AddPictureToTableCell.cs
using System;
using System.Drawing;
using System.Windows.Forms;
using Spire.Doc;
using Spire.Doc.Fields;

namespace AddPictureToTableCell
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            string input = @"..\..\..\..\..\..\Data\TableTemplate.docx";

			//Create a Word document
			Document doc = new Document();

			//Load the file from disk
			doc.LoadFromFile(input);

			//Get the first table from the first section of the document
			Table table1 = (Table)doc.Sections[0].Tables[0];

			//Add a picture to the specified table cell
			DocPicture picture = table1.Rows[1].Cells[2].Paragraphs[0].AppendPicture(Image.FromFile(@"..\..\..\..\..\..\Data\Spire.Doc.png"));

			//Set picture width
			picture.Width = 100;

			//Set picture height
			picture.Height = 100;

			//Save the document
			string output = "AddPictureToTableCell.docx";
			doc.SaveToFile(output, FileFormat.Docx);

			//Dispose the document
			doc.Dispose();
			
            Viewer(output);
        }
        private void Viewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB-Examples/18_Tables/AddPictureToTableCell.vb
Imports Spire.Doc
Imports Spire.Doc.Fields

Namespace AddPictureToTableCell
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			Dim input As String = "..\..\..\..\..\..\Data\TableTemplate.docx"

			'Create a Word document
			Dim doc As New Document()

			'Load the file from disk
			doc.LoadFromFile(input)

			'Get the first table from the first section of the document
			Dim table1 As Table = CType(doc.Sections(0).Tables(0), Table)

			'Add a picture to the specified table cell
			Dim picture As DocPicture = table1.Rows(1).Cells(2).Paragraphs(0).AppendPicture(Image.FromFile("..\..\..\..\..\..\Data\Spire.Doc.png"))
			' =============================================================================
			' Use the following code for netstandard dlls
			' =============================================================================
			'Dim picture As DocPicture = table1.Rows(1).Cells(2).Paragraphs(0).AppendPicture("..\..\..\..\..\..\Data\Spire.Doc.png")
			' =============================================================================

			'Set picture width
			picture.Width = 100

			'Set picture height
			picture.Height = 100

			'Save the document
			Dim output As String = "AddPictureToTableCell.docx"
			doc.SaveToFile(output, FileFormat.Docx)

			'Dispose the document
			doc.Dispose()
			Viewer(output)
		End Sub
		Private Sub Viewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
